srcpkgs/spago/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+build_style="haskell-stack"
+hostmakedepends="stack"
+makedepends="ncurses-libtinfo-devel tar xz zlib-devel"
+short_desc="PureScript package manager & build tool powered by Dhall & package-sets"
+maintainer="Wayne Van Son <waynevanson@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/purescript/spago"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum="3172cb0a5d5f603d77b0b2b6b22652c202417281340743e6478ceef1055d03c9"
+nopie_files="/usr/bin/spago"
+nocross="yes"
```

Comment:
I think the problem here is that `stack` downloads a version of `ghc` that is only for `x86_64`. The template as is should either set `archs="x86_64"` or set `broken` when `"$XBPS_TARGET_MACHINE" != "x86_64"`. (The `archs` variable is really meant to indicate an upstream support limitation, not necessarily a consequence of XBPS packaging details or version incompatibilities.) It's not clear to me what option would be preferable here.

Reply:
> I think the problem here is that stack downloads a version of ghc

Isn't this a problem on its own?

Reply:
> > I think the problem here is that stack downloads a version of ghc
> 
> Isn't this a problem on its own?

Because ideally we want to use an xbps package, right? As I said in a comment, we need to download a specific, non-lts version  of GHC.